src/components/ParserComponent.tsx

```diff
@@ -364,6 +369,9 @@ export default class ParserComponent extends React.Component<ParserInterface,
      * @param {Object} fileEvent: the event passed into this component
      */
     async parse(fileEvent: any) {
+      // clear the console before each parse
+      console.clear();
```

Comment:
The console shouldn't be cleared every time parse is called. It should probably just log a message indicating the start of each parse.

Reply:
ok I will change it